magpylib/_lib/getBvector.py

```diff
@@ -63,13 +64,18 @@ def getBv_magnet(type,MAG,DIM,POSo,POSm,ANG=[],AX=[],ANCH=[]):
     ANCH=[] : length M list of Nx3 numpy arrays float [mm]
         Anchor positions of M subsequent rotations applied ot the N-sized POSm and
         the implicit source orientation.
+    
+    Nphi0=50 : integer gives number of iterations used when calculating diametral
+        magnetized cylindrical magnets.
     """
 
     N = len(MAG)
 
     # set field type
     if type == 'box':
         Bfield = Bfield_BoxV
+    elif type == 'cylinder':
```

Comment:
If this needs to be checked again later in line 96 then this initial check is redundant.
Function input for cylinder also has different format so it may not be a good idea to share a common reference.